analyzer/metadata-plugin/pom.xml

```diff
@@ -77,20 +42,25 @@
                         </goals>
                         <configuration>
                             <minimizeJar>false</minimizeJar>
-                            <keepDependenciesWithProvidedScope>false</keepDependenciesWithProvidedScope>
+                            <keepDependenciesWithProvidedScope>false
+                            </keepDependenciesWithProvidedScope>
```

Comment:
This is another case, where line breaks look super weird after the PR.

Reply:
Fixed